client/benches/tps/utils.rs

```diff
@@ -294,7 +294,7 @@ impl MeasurerUnit {
         let transfer_rose = TransferBox::new(
             asset_id(self.name),
             1_u32.to_value(),
-            asset_id(self.next_name),
+            account_id(self.next_name),
```

Comment:
So the benchmark too was doing a somewhat off thing? I wonder if it would help if we could have the benchmarks integrated in our CI.

Reply:
Yes, basically API was changed, but benchmark is not.
I think it will help a lot to at least run tps oneshot in the CI. 
Because i fixed such errors on multiple occasions already.